internal/service/ec2/vpc_ipam_preview_next_cidr.go

```diff
@@ -0,0 +1,125 @@
+package ec2
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	// "github.com/hashicorp/terraform-provider-aws/internal/flex"
+	// "github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func ResourceVPCIpamPreviewNextCidr() *schema.Resource {
+	return &schema.Resource{
+		Create: ResourceVPCIpamPreviewNextCidrCreate,
+		Read:   ResourceVPCIpamPreviewNextCidrRead,
```

Comment:
```suggestion
		Create: resourceVPCIpamPreviewNextCidrCreate,
		Read:   resourceVPCIpamPreviewNextCidrRead,
```

Reply:
Resolved in 7ac8f4bce5d6b827292ba98098259acd44f7991b.